### PR TITLE
Fro 32 contact us page

### PR DESCRIPTION
**Pull request by olathegreat**
there was a mrge conflict in the previous merge where some components and imported link where affected. the issue has been solved

### CODE REVIEW

File: Apply-For-Me-UI/src/pages/contact_us/ContactUs.jsx [hunk 1]
```diff
@@ -1,7 +1,48 @@
...
+import Footer from "../../components/footer/Footer";
+
 
 const ContactUs = () => {
-    return <div>Conatct us</div>;
+    return (
+        <div className="contact">
+          <div className="contact-nav-wrapper">
+            <Nav/>
+          </div>
```
Comment (Ginohmk) [inline Apply-For-Me-UI/src/pages/contact_us/ContactUs.jsx:14]: - PLease remove the <div className="conttact-nav-wrapper> container. the nav component has its own width.

File: Apply-For-Me-UI/src/pages/contact_us/components/ContactUsForm.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+        
+
+
+                <div className="form-first-line">
+                    <div className="form-group first-line-1">
+                        <label>First Name</label>
+                        <input type="text" required></input>
+
+                    </div>
```
Comment (Ginohmk) [inline Apply-For-Me-UI/src/pages/contact_us/components/ContactUsForm.js:22]: - Work on making your design responsive, your design should stack when the contents are not looking okay as a row

###Your image

image

File: Apply-For-Me-UI/src/pages/contact_us/ContactUs.jsx [hunk 3]
```diff
@@ -1,7 +1,48 @@
...
+            </div>
+            <div className="form-details">
+               <LeftDetails/>
+            </div>
+          </div>
+    
+    
+         <div>
+          <Footer/>
+         </div>
```
Comment (Ginohmk) [inline Apply-For-Me-UI/src/pages/contact_us/ContactUs.jsx:30]: - The footer has its own width, remove parent div

File: Apply-For-Me-UI/src/pages/contact_us/components/ContactStyle.css [hunk 4]
```diff
@@ -0,0 +1,467 @@
...
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+
+}
+.playstore-wrapper img{
+    width: 100px;
+    margin-right: 20px;
+}
+.footer a{
```
Comment (Ginohmk) [inline Apply-For-Me-UI/src/pages/contact_us/components/ContactStyle.css:318]: - Please remove all footer styling. the footer component already has its style.

**Review by Ginohmk — CHANGES_REQUESTED** [review]
Hi @olathegreat , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
you-are-near

## Highlights


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Please work on your responsiveness

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: Apply-For-Me-UI/src/pages/contact_us/components/ContactStyle.css (hunk 4 above)
Comment (olathegreat) [reply Apply-For-Me-UI/src/pages/contact_us/components/ContactStyle.css]: i have worked on every path , do well to check

**Pull request merged**